docker-compose.prod.yml

```diff
@@ -0,0 +1,158 @@
+    entrypoint: [ "sh", "run.sh" ]
+    restart: unless-stopped
+
+  mysql-central:
+    image: mysql:5.6
+    container_name: mysql-central
+    env_file:
+      - ./openmrs-central/prod.db.env
+    ports:
+      - "3307:3306"
+    volumes:
+      - ./openmrs-central/prod.my.cnf:/etc/mysql/my.cnf
+      - openmrsDbData:/var/lib/mysql
+    restart: unless-stopped
+
+  openmrs-central:
+    image: tomcat:7.0.109-jdk8-openjdk-slim
+    container_name: openmrs-central
+    depends_on:
+      - mysql-central
+    volumes:
+      - ./openmrs-central/setenv.sh:/usr/local/tomcat/bin/setenv.sh
+      - ./openmrs-central/run.prod.sh:/usr/local/tomcat/bin/run_openmrs.sh
+      - ./openmrs-central/init.prod.sh:/usr/local/tomcat/bin/init.sh
+      - ./openmrs-central/install_opencr_cert.exp:/usr/local/tomcat/bin/install_opencr_cert.exp
+      - ./openmrs-central/prod-openmrs-runtime.properties:/root/.OpenMRS/openmrs-runtime.properties
+      - ./openmrs-central/openmrs.p12:/root/.OpenMRS/mpi/config/openmrs.p12
+      - ./openmrs-common/modules/legacyui-1.8.2.omod:/root/.OpenMRS/modules/legacyui.omod
+      - ./openmrs-common/modules/owa-1.12.0.omod:/root/.OpenMRS/modules/owa.omod
+      - ./openmrs-common/modules/webservices.rest-2.30.0.omod:/root/.OpenMRS/modules/webservices.rest.omod
+      - ./openmrs-central/modules/debezium-1.0.0-SNAPSHOT.omod:/root/.OpenMRS/modules/debezium.omod
+      - ./openmrs-central/modules/mpi-1.0.0-SNAPSHOT.omod:/root/.OpenMRS/modules/mpi.omod
+      - ./shared/openmrs/tomcat/webapps:/usr/local/tomcat/webapps/  
+      - ./shared/debezium:/root/.OpenMRS/config/debezium
+    ports:
+      - "8080:8080"
+    entrypoint: [ "sh", "/usr/local/tomcat/bin/run_openmrs.sh" ]
+    restart: unless-stopped
+
+  artemis:
+    image: cnocorch/activemq-artemis
+    container_name: artemis
+    ports:
+      - "61616:61616"
+      - "8161:8161"
+    volumes:
+      - ./artemis/artemis-roles.prod.properties:/var/lib/artemis/etc/artemis-roles.properties
+      - ./artemis/artemis-users.prod.properties:/var/lib/artemis/etc/artemis-users.properties
+      - artemisData:/var/lib/artemis
+    restart: unless-stopped
+
+  dbsync-central:
+    image: adoptopenjdk/openjdk8:alpine-slim
+    container_name: dbsync-central
+    env_file:
+      - ./dbsync-central/dbsync.env   
+    environment:
+      - home_dir=/home/eip
+      - shared_dir=/home/eip/shared
+    depends_on:
+      - mysql-central
+      - openmrs-central
+      - artemis
+    ports:
+      - "8081:8081"
+    volumes:
+      - ./dbsync-central/application.prod.properties:/home/eip/application.properties
+      - ./dbsync-central/users.prod.properties:/home/eip/dbsync-users.properties
+      - ./dbsync-central/init.prod.sh:/home/eip/init_eip.sh
+      - ./dbsync-central/run.prod.sh:/home/eip/run.sh
+      - ./dbsync-central/logback-spring.xml:/home/eip/logback-spring.xml
+      - ./shared:/home/eip/shared
```

Comment:
The shared driver needs to be on some sort of cloud drive where you as the engineering team will be publishing new updated builds and shared files.

Reply:
This actually is a folder where some data from the container will be stored to quickly be retrived from the host matchine. I'm talking abaut things like logs, backups generated from the containers, and so on.

Reply:
So where are built jar files and common config properties or templates getting published, because what you want is to have a shared drive between all sites or provinces over the internet where you will be publishing the latest jar and config files and it's where they will be downloaded from, you should not be cloning the repo and building the project

Reply:
Again, this folder has nothing to do with sharing data between sites. This is for sharing some files between containers and host matchine